Source/WebCore/rendering/mac/RenderThemeMac.mm

```diff
@@ -614,6 +618,7 @@ static Color activeButtonTextColor()
 
         case CSSValueButtonface:
         case CSSValueThreedface:
+            // Dark mode is covered in `selectCocoaColor()`.
```

Comment:
Not a big fan of referencing variable / method names in comments that could easily change. But will leave it up to you.

Maybe you could just add an `ASSERT`?